address.event

```diff
@@ -13,7 +13,7 @@
   "queryStringParameters": null,
   "multiValueQueryStringParameters": null,
   "pathParameters": null,
-  "body": "[\n   {\n      \"uprn\": 72262801,\n      \"buildingNumber\": \"8\",\n      \"thoroughfareName\": \"GRANGE FIELDS WAY\",\n      \"postTown\": \"LEEDS\",\n      \"postcode\": \"LS10 4QL\",\n      \"countryCode\": \"GBR\",\n      \"residentFrom\": 1267142400000,\n      \"residentTo\": 1610755200000\n   },\n   {\n      \"uprn\": 63094965,\n      \"buildingNumber\": \"15\",\n      \"dependentLocality\": \"LOFTHOUSE\",\n      \"thoroughfareName\": \"RIDINGS LANE\",\n      \"postTown\": \"WAKEFIELD\",\n      \"postcode\": \"WF3 3SE\",\n      \"countryCode\": \"GBR\",\n      \"residentFrom\": 1610755200000,\n      \"residentTo\": 1627862400000\n   },\n   {\n      \"uprn\": 63042351,\n      \"buildingNumber\": \"5\",\n      \"thoroughfareName\": \"GATEWAYS\",\n      \"postTown\": \"WAKEFIELD\",\n      \"postcode\": \"WF1 2LZ\",\n      \"countryCode\": \"GBR\",\n      \"residentFrom\": 1627862400000,\n      \"currentResidency\": true\n   }\n]",
+  "body": "[\n   {\n      \"uprn\":\"72262801\",\n      \"buildingNumber\": \"8\",\n      \"thoroughfareName\": \"GRANGE FIELDS WAY\",\n      \"postTown\": \"LEEDS\",\n      \"postcode\": \"LS10 4QL\",\n      \"countryCode\": \"GBR\",\n      \"residentFrom\": 1267142400000,\n      \"residentTo\": 1610755200000\n   },\n   {\n      \"uprn\": 63094965,\n      \"buildingNumber\": \"15\",\n      \"dependentLocality\": \"LOFTHOUSE\",\n      \"thoroughfareName\": \"RIDINGS LANE\",\n      \"postTown\": \"WAKEFIELD\",\n      \"postcode\": \"WF3 3SE\",\n      \"countryCode\": \"GBR\",\n      \"residentFrom\": 1610755200000,\n      \"residentTo\": 1627862400000\n   },\n   {\n      \"uprn\": 63042351,\n      \"buildingNumber\": \"5\",\n      \"thoroughfareName\": \"GATEWAYS\",\n      \"postTown\": \"WAKEFIELD\",\n      \"postcode\": \"WF1 2LZ\",\n      \"countryCode\": \"GBR\",\n      \"residentFrom\": 1627862400000,\n      \"currentResidency\": true\n   }\n]",
```

Comment:
What about \"uprn\": 63042351

Reply:
Didnt notice the second one, for some reason it didn't break the test tho 🤔